### PR TITLE
Add WRLinux product WRLinux8 and WRLinux1019 support (V2)

### PR DESCRIPTION
**Pull request by dehuo0**
Description:
Rename wrlinux to wrlinux8 to specify the Wind River Linux product version number.
Since there are serval version Wind River products start to support SCAP e.g. WRLinux8,
WRLinux1019, the supported profiles and functions would be different between these
version products. The old source code for wrlinux in SCAP is just for WRLinux8, so rename
the wrlinux to wrlinux8.
Add the WRLinux1019 platform support, provide the relevant functional files for the WRLinux1019
e.g. profiles, cpe directory etc.

Rationale:
Add home directories for WRLinux8 and WRLinux1019;
Add multi_platform_wrlinux platform to shared.xml files for wrlinux supported rules;
Add wrlinux prodtype to rule.yml files for wrlinux supported rules;
Add multi_platform_wrlinux platform to remediate bash scripts for wrlinux supported rules;

The update in this version pull request is as below:
1. Update the description in CMakeLists.txt to identify WRLinux8 and WRLinux1019 not only WRLinux;
2. Update the installed_OS_is_wrlinux8.xml and installed_OS_is_wrlinux1019.xml to add more version check;

### CODE REVIEW

**Comment by cipherboy** [comment]
@opensca-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci add to white list

**Comment by yuumasato** [comment]
@dehuo0 Thanks for the updated. There was no need for new PR, just pushing the commits to the one already open would suffice.

The changes look good to me.
Now, there are some conflicts, I apologize for this hassle. Just rebasing your branch on top of latest master should resolve them.
Would you like help with that?

**Comment by dehuo0** [comment]
> 
> 
> @dehuo0 Thanks for the updated. There was no need for new PR, just pushing the commits to the one already open would suffice.
> 
> The changes look good to me.
> Now, there are some conflicts, I apologize for this hassle. Just rebasing your branch on top of latest master should resolve them.
> Would you like help with that?

Thank you. I've rebased to the master and updated my patches, please help to check.

File: wrlinux8/templates/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export SHARED_DIR=../../shared/templates/
+export PYTHONPATH=../../shared
+OUTPUT:=output
+OUTPUT_BASH:=$(OUTPUT)/bash
+CSV_DIR:=csv
+
```
Comment (jan-cerny) [inline wrlinux8/templates/Makefile]: I wonder what is the Makefile good for. We don't have makefiles anywehere else than in WRLinux directories. We use CMake for a long time.

File: CMakeLists.txt [hunk 2]
```diff
@@ -241,7 +242,8 @@ message(STATUS "SUSE 12: ${SSG_PRODUCT_SLE12}")
 message(STATUS "Ubuntu 14.04: ${SSG_PRODUCT_UBUNTU1404}")
 message(STATUS "Ubuntu 16.04: ${SSG_PRODUCT_UBUNTU1604}")
 message(STATUS "Ubuntu 18.04: ${SSG_PRODUCT_UBUNTU1804}")
-message(STATUS "WRLinux: ${SSG_PRODUCT_WRLINUX}")
+message(STATUS "WRLinux 8: ${SSG_PRODUCT_WRLINUX8}")
+message(STATUS "WRLinux 1019: ${SSG_PRODUCT_WRLINUX1019}")
```
Comment (jan-cerny) [inline CMakeLists.txt]: Needs to be added in `/build_product` script around line 74.

File: wrlinux8/templates/Makefile (hunk 1 above)
Comment (jan-cerny) [reply wrlinux8/templates/Makefile]: I think the Makefiles can be removed, I can't find any place where they're called.

Comment (dehuo0) [reply wrlinux8/templates/Makefile]: Thanks for the reminding. I will remove the Makefile from WRLinux directories.

File: CMakeLists.txt (hunk 2 above)
Comment (dehuo0) [reply CMakeLists.txt]: Thank you. I will supply these to build_product script.

**Comment by jan-cerny** [comment]
@openscap-ci test this please

**Comment by jan-cerny** [comment]
Thank you for your great contribution!

File: CMakeLists.txt (hunk 2 above)
Comment (yuumasato) [reply CMakeLists.txt]: @jan-cerny Nice catch!

**Comment by dehuo0** [comment]
Why there are so many rules no oval check in RedHat7 profile, for example audit_rules_privileged_commands_*

**Pull request merged**